app/services/ccms/requestors/base_requestor.rb

```diff
@@ -18,6 +18,11 @@ class BaseRequestor
         "xmlns:xsi" => "http://www.w3.org/2001/XMLSchema-instance",
       }.freeze
 
+      CHARACTERS = {
```

Comment:
Could you make this variable name descriptive?

Reply:
I have updated this to SPECIAL_CHARACTER_REPLACEMENTS to try and make this more descriptive.